os/drivers/testcase/test_compress_decompress.c

```diff
@@ -0,0 +1,113 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <tinyara/testcase_drv.h>
+#include <tinyara/binfmt/compression/compress_read.h>
+#include <debug.h>
+#include <errno.h>
+#include <fcntl.h>
+
+/****************************************************************************
+ * Private Declarations
+ ****************************************************************************/
+
+static struct s_header *compression_header;
+static struct s_buffer *buffers;
+
+FAR uint8_t *buffer;
```

Comment:
Please add "static" and remove "FAR".

Reply:
buffers and buffer are not good naming because I can't know what they are.

Reply:
Done. Changed the "buffer" name to "dst_buffer".